### PR TITLE
fix path error when install with luarocks

### PR DESCRIPTION
**Pull request by lilien1010**
### Summary

fix path error when install with luarocks

### Full changelog

* bin/apisix

### CODE REVIEW

File: bin/apisix [hunk 1]
```diff
@@ -103,8 +103,10 @@ worker_shutdown_timeout 3;
 {% if stream_proxy then %}
 stream {
     lua_package_path  "$prefix/deps/share/lua/5.1/?.lua;/usr/share/lua/5.1/apisix/lua/?.lua;]=]
+                      .. [=[/usr/local/apisix/deps/share/lua/5.1/apisix/lua/?.lua;]=]
```
Comment (membphis) [inline bin/apisix]: How about `[=[$prefix/deps/share/lua/5.1/apisix/lua/?.lua;]=]`?

I think this is better.

File: bin/apisix [hunk 2]
```diff
@@ -103,8 +103,10 @@ worker_shutdown_timeout 3;
...
                       .. [=[{*apisix_lua_home*}/lua/?.lua;;{*lua_path*};";
     lua_package_cpath "$prefix/deps/lib64/lua/5.1/?.so;]=]
+                      .. [=[/usr/local/apisix/deps/lib/lua/5.1/?.so;]=]
```
Comment (membphis) [inline bin/apisix]: I do not think this line is useful. We should drop it.

Comment (lilien1010) [reply bin/apisix]: understood

File: bin/apisix [hunk 3]
```diff
@@ -149,8 +150,10 @@ stream {
 
 http {
     lua_package_path  "$prefix/deps/share/lua/5.1/?.lua;/usr/share/lua/5.1/apisix/lua/?.lua;]=]
+                      .. [=[/usr/local/apisix/deps/share/lua/5.1/apisix/lua/?.lua;]=]
```
Comment (membphis) [inline bin/apisix]: should use `$prefix` too.

File: bin/apisix [hunk 4]
```diff
@@ -149,8 +150,10 @@ stream {
...
                       .. [=[{*apisix_lua_home*}/lua/?.lua;;{*lua_path*};";
     lua_package_cpath "$prefix/deps/lib64/lua/5.1/?.so;]=]
+                      .. [=[/usr/local/apisix/deps/lib/lua/5.1/?.so;]=]
```
Comment (membphis) [inline bin/apisix]: useless line, please remove it

File: bin/apisix (hunk 3 above)
Comment (lilien1010) [reply bin/apisix]: sorry, remove is done now

File: bin/apisix (hunk 4 above)
Comment (lilien1010) [reply bin/apisix]: sorry, remove is done now

**Pull request merged**